packages/amp/src/processors/removeProhibitedTags.ts

```diff
@@ -0,0 +1,19 @@
+import { Processor, Element } from "@frontity/html2react/types";
+import { Packages } from "../../types";
+
+export const removeProhibitedTags: Processor<Element, Packages> = {
+  name: "amp: removeProhibitedTags",
+  test: ({ node }) =>
+    node.type === "element" &&
+    (node.component === "style" ||
+      node.component === "script" ||
+      node.component === "picture" ||
```

Comment:
Do you know if for `picture` we should return the children `img` tag to honor the fallback or that is not necessary?

Docs: https://www.w3schools.com/tags/tag_picture.asp

```html
<picture>
  <source media="(min-width:650px)" srcset="img_pink_flowers.jpg" />
  <source media="(min-width:465px)" srcset="img_white_flower.jpg" />
  <img src="img_orange_flowers.jpg" alt="Flowers" style="width:auto;" />
</picture>
```

Reply:
I think we could do that. I would however also throw a warning saying that this is happening so that the user could switch from `<picture>` to `<img>` in their code.

Reply:
While working on this, I've found this bug: https://github.com/frontity/frontity/issues/742 

It's not strictly a blocker but I don't think we should merge until it's solved because if there is a `style` attribute on `img`,`iframe`, `audio`, etc. the AMP markup will not be correct.

Reply:
Good catch. I answered in the issue 🙂